ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/control/JogControlTopComponent.java

```diff
@@ -28,18 +28,19 @@ This file is part of Universal Gcode Sender (UGS).
 import org.openide.awt.ActionID;
 import org.openide.awt.ActionReference;
 import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
 
 /**
- * Top component which displays something.
+ * Old jog controller
+ *
+ * @deprecated This jog controller is deprecated and should be replaced by the new jog module.
  */
 @TopComponent.Description(
         preferredID = "JogControlTopComponent",
         //iconBase="SET/PATH/TO/ICON/HERE", 
         persistenceType = TopComponent.PERSISTENCE_ALWAYS
 )
-@TopComponent.Registration(mode = "middle_left", openAtStartup = true)
-@ActionID(category = LocalizingService.JogControlCategory, id = LocalizingService.JogControlActionId)
-@ActionReference(path = LocalizingService.JogControlWindowPath)
+@TopComponent.Registration(mode = "middle_left", openAtStartup = false)
 @TopComponent.OpenActionRegistration(
```

Comment:
This annotation creates the actions in the Window menu, so they should be removed too. Removing old actions has been a big challenge for me, I think you're supposed to do it in `layer.xml`, there are a few examples here:

./ugs-platform/ugs-platform-ugscore/src/main/resources/com/willwinder/ugs/nbp/core/layer.xml

Details here: http://wiki.netbeans.org/NbmIdioms#XML_Layers